plenum/server/node.py

```diff
@@ -2109,6 +2109,7 @@ def on_view_change_complete(self, view_no):
 
     def start_catchup(self):
         # Process any already Ordered requests by the replica
+        # TODO: What if catchup already is started? Should we handle it?
```

Comment:
I think we should check for ledger's state here to prevent multiple catchup start